airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/ReplicationActivityImpl.java

```diff
@@ -169,19 +171,26 @@ public StandardSyncOutput replicate(final JobRunConfig jobRunConfig,
   }
 
   private static StandardSyncOutput reduceReplicationOutput(final ReplicationOutput output) {
-    final long totalBytesReplicated = output.getReplicationAttemptSummary().getBytesSynced();
-    final long totalRecordsReplicated = output.getReplicationAttemptSummary().getRecordsSynced();
-
+    final StandardSyncOutput standardSyncOutput = new StandardSyncOutput();
     final StandardSyncSummary syncSummary = new StandardSyncSummary();
-    syncSummary.setBytesSynced(totalBytesReplicated);
-    syncSummary.setRecordsSynced(totalRecordsReplicated);
-    syncSummary.setStartTime(output.getReplicationAttemptSummary().getStartTime());
-    syncSummary.setEndTime(output.getReplicationAttemptSummary().getEndTime());
-    syncSummary.setStatus(output.getReplicationAttemptSummary().getStatus());
-    syncSummary.setTotalStats(output.getReplicationAttemptSummary().getTotalStats());
-    syncSummary.setStreamStats(output.getReplicationAttemptSummary().getStreamStats());
+    final ReplicationAttemptSummary replicationSummary = output.getReplicationAttemptSummary();
+
+    syncSummary.setBytesSynced(replicationSummary.getBytesSynced());
+    syncSummary.setRecordsSynced(replicationSummary.getRecordsSynced());
+    syncSummary.setStartTime(replicationSummary.getStartTime());
+    syncSummary.setEndTime(replicationSummary.getEndTime());
+    syncSummary.setStatus(replicationSummary.getStatus());
+    syncSummary.setTotalStats(replicationSummary.getTotalStats());
+    syncSummary.setStreamStats(replicationSummary.getStreamStats());
+
+    final SyncStats totalStats = syncSummary.getTotalStats();
+    totalStats.setReplicationStartTime(totalStats.getReplicationStartTime());
+    totalStats.setReplicationEndTime(totalStats.getReplicationEndTime());
+    totalStats.setSourceReadStartTime(totalStats.getSourceReadStartTime());
+    totalStats.setSourceReadEndTime(totalStats.getSourceReadEndTime());
+    totalStats.setDestinationWriteStartTime(totalStats.getDestinationWriteStartTime());
+    totalStats.setDestinationWriteEndTime(totalStats.getDestinationWriteEndTime());
```

Comment:
I'm not sure I understand this bit - why are we setting the same values again?

Reply:
There's a nested object thing going on here - the replication activity has stats that we are passing up to the attempt stats

Reply:
Hmm... but we're both getting and setting from the same `totalStats` object, no?

Reply:
If this is working, I think the `syncSummary.setTotalStats(replicationSummary.getTotalStats());` line above covers this and the `totalStats.setDestinationWriteEndTime(totalStats.getDestinationWriteEndTime());` lines are essentially no-ops

Reply:
.... I think you are right! I'll test that now. 🤯

Reply:
Yep.  Sorted via bd0d8927d8d2e144515183192825ac11224f967a